File: app/src/main/java/org/baole/oned/util/markdown/InlineParserFactory.java
package org.baole.oned.util.markdown;

import org.commonmark.parser.InlineParserContext;

public class InlineParserFactory implements org.commonmark.parser.InlineParserFactory {
    @Override
    public org.commonmark.parser.InlineParser create(InlineParserContext inlineParserContext) {
        return new InlineParser(inlineParserContext.getCustomDelimiterProcessors());
    }
}

File: app/src/main/java/org/baole/oned/util/markdown/InlineParserFactory.kt
package org.baole.oned.util.markdown

import org.commonmark.parser.InlineParserContext
import org.commonmark.parser.InlineParserFactory

class InlineParserFactory : InlineParserFactory {
    override fun create(inlineParserContext: InlineParserContext): org.commonmark.parser.InlineParser {
        return InlineParser(inlineParserContext.customDelimiterProcessors)
    }
}